spec/unit/models/project_spec.rb

```diff
@@ -40,4 +40,51 @@
       end
     end
   end
+
+  describe '#blueprint' do
+    let(:project_data) do
+      { 'project' => {
+          'links' => { 'self' => '/gdc/foo' },
+          'meta' => { 'title' => 'My Project' } } }
+    end
+    let(:diff_result) do
+      { 'asyncTask' => { 'link' => { 'poll' => 'poll_me' } } }
+    end
+    let(:blueprint) { double(GoodData::Model::ProjectBlueprint) }
+    subject { GoodData::Project.new(project_data) }
+    before do
+      allow(client).to receive(:get).and_return(diff_result)
+      allow(client).to receive(:poll_on_code)
+      allow(GoodData::Model::FromWire).to receive(:from_wire) .and_return(blueprint)
+      allow(blueprint).to receive(:title=)
+      subject.client = client
+    end
+
+    context 'when include_ca option is true' do
+      it 'requests model view with includeCA=true' do
+        expect(client).to receive(:get).with('/gdc/projects/foo/model/view',
+                           hash_including(params:
+                             hash_including(includeCA: true)))
+        subject.blueprint(include_ca: true)
+      end
+    end
+
+    context 'when include_ca option is false' do
+      it 'requests model view with includeCA=false' do
+        expect(client).to receive(:get).with('/gdc/projects/foo/model/view',
+                           hash_including(params:
```

Comment:
Style/AlignParameters: Align the parameters of a method call if they span more than one line.